CMakeLists.txt

```diff
@@ -572,7 +572,7 @@ function(add_picolibc directory variant target_triple flags qemu_params variant_
 
     function(picolibc_test test)
         set(BUILD_PARAMS --config ${variant}_semihost.cfg
-                -T ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/tests/ldscripts/picolibc_${variant}.ld
+                -T ${CMAKE_CURRENT_FUNCTION_LIST_DIR}/test-support/ldscripts/picolibc_${variant}.ld
```

Comment:
Just a note: the ld files will be removed with my change to run picolibc tests with meson.
If this change goes first, it can stay as is.